RELEASE_NOTES.md

```diff
@@ -1,153 +1,136 @@
-checking is off.
-
-## Multi-package Support
-The `package` provider has been extended to support multiple packages. This
-support is new and and not all subproviders yet support it. Full support for
-`apt` and `yum` has been implemented.
+```
+> Get-DscResource User
 
-## chef_gem deprecation of installation at compile time
+ImplementedAs   Name                      Module                         Properties
+-------------   ----                      ------                         ----------
+PowerShell      User                      PSDesiredStateConfiguration    {UserName, DependsOn, Descr...
 
-A `compile_time` flag has been added to the chef_gem resource to control if it is installed at compile_time or not.  The prior behavior has been that this
-resource forces itself to install at compile_time which is problematic since if the gem is native it forces build_essentials and other dependent libraries
-to have to be installed at compile_time in an escalating war of forcing compile time execution.  This default was engineered before it was understood that a better
-approach was to lazily require gems inside of provider code which only ran at converge time and that requiring gems in recipe code was bad practice.
+```
 
-The default behavior has not changed, but every chef_gem resource will now emit out a warning:
+We see here that is `ImplementedAs` is not equal to `Composite`, so it is a resource that can
+be used with `dsc_resource`. We can what properties are accpeted by the `User` resource by
+running
 
 ```
-[2015-02-06T13:13:48-08:00] WARN: chef_gem[aws-sdk] chef_gem compile_time installation is deprecated
-[2015-02-06T13:13:48-08:00] WARN: chef_gem[aws-sdk] Please set `compile_time false` on the resource to use the new behavior.
-[2015-02-06T13:13:48-08:00] WARN: chef_gem[aws-sdk] or set `compile_time true` on the resource if compile_time behavior is required.
+> Get-DscResource User | Select-Object -ExpandProperty Properties
+
+Name                      PropertyType                            IsMandatory Values
+----                      ------------                            ----------- ------
+UserName                  [string]                                       True {}
+DependsOn                 [string[]]                                    False {}
+Description               [string]                                      False {}
+Disabled                  [bool]                                        False {}
+Ensure                    [string]                                      False {Absent, Present}
+FullName                  [string]                                      False {}
+Password                  [PSCredential]                                False {}
+PasswordChangeNotAllowed  [bool]                                        False {}
+PasswordChangeRequired    [bool]                                        False {}
+PasswordNeverExpires      [bool]                                        False {}
 ```
```

Comment:
For a while, that was the only real docs that there were that you could trust on a resource. :)